x-pack/plugins/maps/public/classes/styles/color_palettes.test.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import {
+  getColorRampCenterColor,
+  getOrdinalMbColorRampStops,
+  getColorPalette,
+} from './color_palettes';
+
+jest.mock('ui/new_platform');
```

Comment:
You can remove this line. Probably copied from another test where it should also be removed.